treeherder/model/derived/jobs.py

```diff
@@ -431,6 +431,20 @@ def get_performance_series_summary(self, interval_seconds):
 
         return series_summary
 
+    def get_platforms(self, interval_seconds):
```

Comment:
Oh, sorry. I know what's your mean. but somehow change the wrong place. Sorry about that. :(